tests/api/test_acl_counter.py

```diff
@@ -19,23 +22,33 @@ def test_acl_counter_create(self, npu):
                 'attributes': ['SAI_ACL_COUNTER_ATTR_TABLE_ID', '$acl_table_1'],
             },
         ]
-
+        npu.objects_discovery()
```

Comment:
why do we need it here? looks like you do not use discovered objects

Reply:
this was a type , resolved this issue